android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationPageSettings.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.translate
+
+/**
+ * Translation settings that relate to the page
+ *
+ * @property alwaysOfferPopup The setting for if translations should automatically be offered.
+ * When true, on detected translatable pages different from the user's preferred languages,
+ * the engine will offer to translate the page.
+ * @property alwaysTranslateLanguage The setting for if the current page language should be
```

Comment:
```suggestion
 * @property alwaysTranslateLanguage The setting for whether the current page language should be
```
This suggestion applies for all of the other @property